cases/deposit.test.js

```diff
@@ -29,15 +31,15 @@ describe("Deposit", () => {
       },
     });
     infoJSON = await infoResponse.json();
-    const currencies = Object.keys(infoJSON.deposit);
-    enabledCurrency = currencies.find(
-      (currency) => infoJSON.deposit[currency].enabled,
-    );
+    currencies = Object.keys(infoJSON.deposit);
+    enabledCurrency = testCurrency
+      ? testCurrency
+      : currencies.find((currency) => infoJSON.deposit[currency].enabled);
     ({ token: jwt } = await getSep10Token(url, keyPair));
   });
 
   it("has a currency enabled for deposit", () => {
-    expect(enabledCurrency).toEqual(expect.any(String));
+    expect(currencies).toEqual(expect.arrayContaining([enabledCurrency]));
```

Comment:
We need to make sure the currency is actually enabled for deposit, not just that it exists in the dictionary

Reply:
But doesn't that dictionary represent the currencies enabled for deposit?

Reply:
Oh, it could have a "enabled" false, right?

Reply:
yeah thats what i meant, make sure its there, and that its enabled:true